src/ui/components/login/login_form.rs

```diff
@@ -28,6 +28,7 @@ pub async fn login(username_or_email: String, password: String) -> Result<(), Se
       session.insert("jwt", jwt.into_inner())?;
     }
 
+    redirect("/");
```

Comment:
This is taken care of in the isomorphic effect. I can't do it in this server function because I need to refetch the site response, which seems to only be doable in the component.

Reply:
is required for SSR only login, the effect is not triggered